PWGHF/DataModel/HFSecondaryVertex.h

```diff
@@ -611,7 +611,7 @@ DECLARE_SOA_COLUMN(FlagMCDecayChanGen, flagMCDecayChanGen, int8_t); //! resonant
 // mapping of decay types
 enum DecayType { DPlusToPiKPi = 0,
                  LcToPKPi,
-                 DsToPiKK,
+                 DsToKKPi,
```

Comment:
Why?

Reply:
It's not a big deal but I am still wondering what is the reason for this change.